utils/gyb_syntax_support/Trivia.py

```diff
@@ -62,13 +62,13 @@ def is_collection(self):
 ]
 
 
-def verify_no_duplicate_serialization_codes(trivias):
+def verify_no_duplicate_serialization_codes(trivia):
```

Comment:
I would prefer to continue using `trivias` here to match the global `TRIVIAS` variable, even though it’s not a valid English pluralization. It also works around the problem to rename `trivia` to `item` below.